modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -188,15 +188,7 @@ This window may be several hours after the client is registered.
 To speed up {micro} registration, manually reboot the client after the registration script finishes.
 ====
 
-To register your {micro} clients, you need a bootstrap repository.
-By default, bootstrap repositories are automatically created, and regenerated daily for all synchronized products.
-You can manually create the bootstrap repository from the command prompt, using this command:
-
-----
-mgr-create-bootstrap-repo
-----
-
-For more information on registering your clients, see xref:client-configuration:registration-overview.adoc[].
+include::snippets/create_bootstrap_repo_register.adoc[]
```

Comment:
I don't think this is the only place where we have this text? Probably we want to use the snippet in more places. Use grep. I found a lot of other adoc files.

Reply:
ok, thanks, it is ready for review now.